README.md (#1)
LucasGobatto/WeatherApp: README.md

WeatherApp

- Project's Description

>   This project has the proposal to create a weather app. With the app, users can get informations about

Reviewer: ```suggestion
  This project's proposal is to create a weather app. With the app, users can get information about
```

>   the climate of them places.

Reviewer: ```suggestion
  the climate on their location or any other.
```

- Environment and Tools

>   node.js v12.14.0

Reviewer: Add yarn (it's a bit faster than npm)

  xcode v11.7
  VSCode v1.49.0

- Steps to run and debug

  - Web:

  - Mobile:
    - On project folder:
      - install dependences: npm i
      - on ios folder: pod install

>       - start the bandle with: npx react-native start

Reviewer: ```suggestion
      - start the bundle with: npx react-native start
```

>       - run ios projetc with: npx react-native run-ios

Reviewer: ```suggestion
      - run ios project with: npx react-native run-ios
```
